README.md

```diff
@@ -25,6 +25,59 @@ This is not designed to be realtime. Postoffice uses [GenStage](https://github.c
 * Cluster nodes. Best effort to avoid sending messages more than once. (More info on clustering later)
 * Endpoint to be used as health-check from k8s `/api/health`
 
+## API
+We expose an API to enable projects to create the structure they need to work: topics, publishers and messages.
+For both topics and publishers, if the resource already exist we return `400 bad request`.
+
+### Topics
+Here we have a sample request to create a topic
+```
+POST /api/topics
+{
+  "name": "example-topic",
+  "origin_host": sender_service.com
+}
+```
+Attributes:
+* _name_: topic name, it's what publishers will use to subscribe.
+* _origin_host_: Host URL where messages for this topic will come from. This will used in a future feature to recover undelivered messages.
+
+### Publishers
+Publishers creation example:
+```
+POST /api/publishers
+{
+  "active": True,
+  "topic": "example-topic",
+  "target": "http://myservice.com/examples",
+  "type": "http/pubsub",
+  "from_now": True
+}
+```
+Attributes:
+* _active_: if this publisher is active and must check and send pending messages or not.
+* _topic_: from which topic this publisher should take messages.
+* _target_: the endpoint where the post request with the message will be done.
+* _type_: We now support two different types of publishers.
+  * _http_: the messages are sent via POST request to a target url.
+  * _pubsub_: messages are published to a target topic on GCloud Pub/Sub service.
+* _from_now_: This param controls if you want to receive messages from this topic from now or in the other hand you're interested in the complete topic's messages.
+
+### Messages
+Message creation example:
+```
+POST /api/messages
+{
+  "topic": topic,
+  "payload": {},
```

Comment:
topic is a required field, payload I should to check if is required

Reply:
Empty messages are not allowed